<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601593287106' stop='1601593381014'>
    <name>Paid blocks</name>
    <title>Paid blocks</title>
    <test-cases>
        <test-case start='1601593292440' status='failed' stop='1601593320097'>
            <name>MailChimp Block</name>
            <title>MailChimp Block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601593292441' status='undefined'>
                    <name>Can visit the block editor and add a MailChimp block</name>
                    <title>Can visit the block editor and add a MailChimp block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: MailChimp Block' source='0ce59d33-2c65-40c0-80b4-17966415fa97-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1601593320097' status='failed' stop='1601593347721'>
            <name>Pay with PayPal</name>
            <title>Pay with PayPal</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601593320098' status='undefined'>
                    <name>Can visit the block editor and add a Pay with PayPal block</name>
                    <title>Can visit the block editor and add a Pay with PayPal block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: Pay with PayPal' source='72046d4f-4e62-4303-aa29-f5fc59e8f51a-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1601593347721' status='failed' stop='1601593375294'>
            <name>WordAds block</name>
            <title>WordAds block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601593347722' status='undefined'>
                    <name>Can visit the block editor and add a WordAds block</name>
                    <title>Can visit the block editor and add a WordAds block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: WordAds block' source='1a0f657c-d4d7-47f9-a3da-b418257dd248-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>